core/block/object/objectcreator/chat.go

```diff
@@ -4,28 +4,48 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/globalsign/mgo/bson"
 	"github.com/gogo/protobuf/types"
 
 	"github.com/anyproto/anytype-heart/core/block/editor/state"
 	"github.com/anyproto/anytype-heart/core/domain"
 	"github.com/anyproto/anytype-heart/pkg/lib/bundle"
+	smartblock2 "github.com/anyproto/anytype-heart/pkg/lib/core/smartblock"
 	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
 	"github.com/anyproto/anytype-heart/space/clientspace"
 	"github.com/anyproto/anytype-heart/util/pbtypes"
 )
 
 func (s *service) createChat(ctx context.Context, space clientspace.Space, details *types.Struct) (string, *types.Struct, error) {
+	key, err := domain.NewUniqueKey(smartblock2.SmartBlockTypeChatDerivedObject, bson.NewObjectId().Hex())
```

Comment:
derivation makes no sense with random ids

Reply:
that was typo